packages/dev/core/src/Culling/ray.ts

```diff
@@ -925,6 +925,12 @@ Scene.prototype.pickWithBoundingInfo = function (
     return result;
 };
 
+Object.defineProperty(Scene.prototype, "_pickingAvailable", {

```

Comment:
Why not using a constant instead of a property ? looks like this is a static value so only defined if this file is imported.

Reply:
Also it looks like this could be part of input manager more than scene.